cmd/short/short.go

```diff
@@ -10,26 +10,57 @@ import (
 	"net/http"
 	"net/url"
 	"bytes"
+	//"go/build"
+	//"os"
+	"sync"
 )
 
 const CleanUrl_api string = "https://cleanuri.com/api/v1/shorten"
 const Relink_api string = "https://rel.ink/api/links/"
 
 var flagRelink *bool = flag.Bool("relink", false, "a bool")
 
-type CleanApiAnswer struct {
-	Result_url string
+type CleanUrlAnswer struct {
+	ResultUrl  string `json:"result_url"`
 	Error      string
 }
 
+// Wrapper for CleanUrl() and Relink()
+func RunInParallel(a func(url_link string) (string, error), args []string) map[string]error {
+	shorts := make(map[string]error)
+	var mutex = &sync.Mutex{}
+	var wg sync.WaitGroup
+	var short string
```

Comment:
There are no need to declare `short` and `err` variables before using them

Reply:
memory allocation in for cycle?